.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: go
+
+go:
+  - "1.9"
```

Comment:
Use 1.x instead

Reply:
cvefeed/cvefeed.go and cvefeed/dictionary.go use type aliases which is >= 1.9 feature.

Reply:
Deploy what? All the binaries from cmd/...? I believe it is unnecessary; until we have proper packaging (rpms, debs etc.) it's better to stick with manual `git install ./...`